db/model/Variable.ts

```diff
@@ -181,3 +182,39 @@ export async function writeVariableCSV(
         row[variableColumnIndex[datum.variableId]] = datum.value
     }
 }
+
+export const getDataValue = async ({
+    variableId,
+    entityId,
+    year,
+}: DataValueQueryArgs) => {
+    if (!variableId || !entityId) return
+
+    const queryStart = `
+        SELECT value, year, variables.unit as unit, entities.name as entityName FROM data_values
+        JOIN entities on entities.id = data_values.entityId
+        JOIN variables on variables.id = data_values.variableId
+        WHERE entities.id = ?
+        AND variables.id= ?`
```

Comment:
If we want the value formatting to match the way the Grapher formats values, we'll also need to consider the `variables.display` configuration.  

E.g. `variables.unit` and `variables.shortUnit` columns tend to only be defined in exceptional cases, when the imported dataset has a unit defined at import time (which only happens for bulk imports).

Mostly, the units are defined in `variables.display->"$.unit"`. This is what the authors have control over through the OWID Admin interface, and the Grapher will use a `display` property, if available, and only then default to the `variables.unit` field. 

The `display` JSON field on the `variables` table has this schema: https://github.com/owid/owid-grapher/blob/9bc2033bbdc0e834d0d8589a080232d602e6f84e/grapher/core/LegacyVariableCode.ts#L27-L41

There are some additional transformations happening:
- `conversionFactor: number` multiplies the value, e.g. if the original imported dataset has a value `1000 grams`, you can set `{ conversionFactor: 0.001, unit: "kg" }` in order to turn it into `1 kg`. 
- `yearIsDay: boolean` means the "year" actually needs to be formatted as a day. The year value is actually an offset (in days) from `zeroDay: string`. 
- `numDecimalPlaces` would round the value to the specified decimal places

If this is just a very limited test, we can probably get away without these formatting issues.

Reply:
Maybe one easy extension is to pass the value through `formatValue()`, which implements at least some of these transformations: https://github.com/owid/owid-grapher/blob/95504e57f04a3e9dd9e82c78ec59475814f752bc/clientUtils/formatValue.ts#L16-L81

Reply:
@danielgavrilov thanks for the background, that's very helpful. I gave `formatValue` a go but it didn't go far enough given what options I could pass it at this stage. 

I'll set formatting aside for now, as the immediate experiment focuses on piloting charts. The "pulling data from the db" part is more incidental at this point and can be manual.